src/main/kotlin/biokotlin/cli/ValidateAndCorrectGVCFs.kt

```diff
@@ -0,0 +1,115 @@
+package biokotlin.cli
+
+import biokotlin.genome.fastaToNucSeq
+import biokotlin.util.bufferedReader
+import biokotlin.util.bufferedWriter
+import biokotlin.util.parseVCFFile
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import kotlinx.coroutines.runBlocking
+import java.io.File
+
+class ValidateAndCorrectGVCFs : CliktCommand(help = "Validate and correct GVCF files") {
+
+    val inputDir by option(help = "Full path to input GVCF file directory")
+        .required()
+
+    val outputDir by option(help = "Full path to output GVCF file directory")
+        .required()
+
+    val referenceFile by option(help = "Full path to reference fasta file")
+        .required()
+
+    override fun run() {
+
+        // Checks to ensure that the input and output directories and reference file exist
+        if (!File(inputDir).isDirectory) {
```

Comment:
Consider using Kotlin's require statements here.

Reply:
Ok, I changed that